apps/studio/components/interfaces/Realtime/Inspector/Header.tsx

```diff
@@ -41,25 +40,13 @@ export const Header = ({ config, onChangeConfig }: HeaderProps) => {
             >
               {config.enabled ? `Stop listening` : `Start listening`}
             </Button>
-          </Tooltip.Trigger>
+          </TooltipTrigger_Shadcn_>
           {config.channelName.length === 0 && (
-            <Tooltip.Portal>
-              <Tooltip.Content side="bottom">
-                <Tooltip.Arrow className="radix-tooltip-arrow" />
-                <div
-                  className={[
-                    'rounded bg-alternative py-1 px-2 leading-none shadow',
-                    'border border-background',
-                  ].join(' ')}
-                >
-                  <span className="text-xs text-foreground">
-                    You'll need to join a channel first
-                  </span>
-                </div>
-              </Tooltip.Content>
-            </Tooltip.Portal>
+            <TooltipContent_Shadcn_ side="bottom">
+              You'll need to join a channel first
```

Comment:
I don't think I can see this tooltip

Reply:
It's shown if you hover over the "start listening" without setting a channel.